src/js/components/Select/SelectOption.js

```diff
@@ -1,42 +1,39 @@
-import React, { Component } from 'react';
+import React from 'react';
 
 import { Box } from '../Box';
 import { Button } from '../Button';
 import { withForwardRef } from '../hocs';
 
-class SelectOption extends Component {
-  shouldComponentUpdate(nextProps) {
-    const { active, disabled, option, selected } = this.props;
-    const {
-      active: nextActive,
-      disabled: nextDisabled,
-      option: nextOption,
-      selected: nextSelected,
-    } = nextProps;
-    return (
-      active !== nextActive ||
-      selected !== nextSelected ||
-      disabled !== nextDisabled ||
-      option !== nextOption
-    );
-  }
-
-  render() {
-    const { forwardRef, ...rest } = this.props;
-    return (
-      <Box flex={false}>
-        <Button
-          tabIndex="-1"
-          ref={forwardRef}
-          role="menuitem"
-          hoverIndicator="background"
-          {...rest}
-        />
-      </Box>
-    );
-  }
+function shouldComponentUpdate(prevProps, nextProps) {
+  const { active, disabled, option, selected } = prevProps;
+  const {
+    active: nextActive,
+    disabled: nextDisabled,
+    option: nextOption,
+    selected: nextSelected,
+  } = nextProps;
+  return (
```

Comment:
`React.memo` actually takes `areEqual` as second argument. It is the opposite of `shouldComponentUpdate`.
Reference: https://reactjs.org/docs/react-api.html#reactmemo
```suggestion
return (
  active === nextActive &&
  selected === nextSelected && 
  disabled === nextDisabled && 
  option === nextOption 
  );
```
You can also update the function name to match the context here.

Reply:
Thanks. Fixed it.